layers/+lang/python/packages.el

```diff
@@ -110,7 +110,24 @@
 
 (defun python/init-hy-mode ()
   (use-package hy-mode
-    :defer t))
+    :defer t
+    :init
+    (progn
+      (add-hook 'hy-mode-hook 'paredit-mode)
```

Comment:
Why `paredit-mode` and not `smartparens-mode`? Either way, the user can exclude both Paredit and Smartparens, perhaps use `with-eval-after-load` or move to `python/post-init-smartparens`?